### PR TITLE
Reorganize

### PR DESCRIPTION
**Pull request by jbae11**
This pr does:
* organize test files
* separate test files from example input files
* pep8 issues for all python files (didn't solve E501 -line too long)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,8 +51,8 @@ jobs:
                 command: |
                     pip install -U pytest
                     python setup.py install
-                    pytest ./tests/solver_unit_tests.py
-                    pytest ./tests/no_inst_fleet_unit_tests.py
+                    pytest ./tests/unit_tests/solver_unit_tests.py
```
Comment (gwenchee) [inline .circleci/config.yml]:
```suggestion
                    pytest ./tests/unit_tests/deploy_solver_unit_tests.py
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,8 +51,8 @@ jobs:
...
+                    pytest ./tests/NO_fleet_unit_tests.py
```
Comment (gwenchee) [inline .circleci/config.yml]:
```suggestion
                    pytest ./tests/integration_unit_tests/NO_fleet_unit_tests.py
```

**Review by gwenchee — CHANGES_REQUESTED** [review]
Other than the two comments, I think it looks good.

**Comment by gwenchee** [comment]
This PR addresses https://github.com/ergs/d3ploy/issues/123

**Review by FlanFlanagan — COMMENTED** [review]
Seems like mostly pep8 stuff. 

But the changes look good to me.  I think we can merge when it passes tests.

**Pull request merged**